### PR TITLE
MDC 0.30.0

### PR DESCRIPTION
**Pull request by j-o-d-o**
#139

### CODE REVIEW

File: src/Select/index.js [hunk 1]
```diff
@@ -67,10 +80,9 @@ type SelectPropsT = {
   /** Placeholder text for the form control. */
   placeholder?: string,
   /** Disables the form control. */
-  disabled?: boolean,
-  /** Makes a cssOnly select. */
-  cssOnly?: boolean
```
Comment (izakfilmalter) [inline src/Select/index.js]: RIP

Comment (jamesmfriedman) [reply src/Select/index.js]: Hahaha. I could tell by your cod examples you loves them. Shouldn’t be hard to make a wrapper for.

Comment (izakfilmalter) [reply src/Select/index.js]: It's more that menu has issues when inside something that is transformed or has overflow hidden. Which is like 3/4 of my mobile interface.

**Comment by jamesmfriedman** [comment]
Alright, looks like we have a release. Thanks for you help @j-o-d-o and @izakfilmalter. Merging to master and doing one last check before the deploy.

**Pull request merged**